jinja_utils_test.py

```diff
@@ -108,3 +119,14 @@ def test_evaluate_object(self):
         parsed_dict = jinja_utils.evaluate_object(orig_dict, {'b': 'c'})
         self.assertEqual(orig_dict, {'a': '{{b}}'})
         self.assertEqual(parsed_dict, {'a': 'c'})
+
+        # int type input used.
+        parsed_object = jinja_utils.evaluate_object(34, {})
+        self.assertEqual(parsed_object, 34)
+
+    def test__log2_floor_filter(self):
+        log_value = jinja_utils._log2_floor_filter(10)
```

Comment:
```_log2_floor_filter(10)``` is a protected member of class. Ditto below.

Reply:
@WickedBrat
Thanks. 
I know it's protected.is there any other way to call that function.
https://stackoverflow.com/a/35701863/5707801

Reply:
I think it's OK to do it in this case, but make sure to use the human-readable pylint-disable warning and not the numeric code. (See other parts of the codebase that do this.)

Thanks!

Reply:
Done..!

Reply:
Another way to unittest protected members without the linting error is to do them as a [`doctest`](https://docs.python.org/2/library/doctest.html).

But noticeably, the repo doesn't use doctests anywhere currently.(so maybe they should be considered in future @seanlip ?)

Reply:
Thanks for the suggestion @nalinbhardwaj! I have no strong opinion here. I'm going to defer to @apb7's decision on that, since he leads this part of the project!

Reply:
Hi @nalinbhardwaj, thanks for the suggestion. I think it's a bit more organized to keep all tests in a separate test file (as we currently have) instead of testing some methods using doctest and others using the conventional way. Therefore, we can continue to use pylint pragmas for a single line to test protected methods without much problem.